src/pfs/pfs.proto

```diff
@@ -376,7 +376,16 @@ message ListFileRequest {
   // repo, the commit/branch, and path prefix of files we're interested in
   // If the "path" field is omitted, a list of files at the top level of the repo
   // is returned
-  File file = 1;
+  File parentDirectory = 1;
```

Comment:
Changing the name of this field would be a breaking change and this isn't an accurate name regardless of the above comment. This field can refer to a regular file.

Reply:
I didn't realize this would be a breaking change. I get your point about the name not being accurate though. I've reverted the name change